version/version.go

```diff
@@ -1,4 +1,4 @@
 package version
 
 // Version is the version of the build.
-const Version = "0.57.6-dev"
+const Version = "0.57.8-dev"
```

Comment:
Shouldn’t the versions be `0.57.6` released, `0.57.7-dev` committed?

Skipping 0.57.6 wouldn’t really hurt, it would just be a bit surprising.